CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 - [#353](https://github.com/pusher/oauth2_proxy/pull/353) Fix login page fragment handling after soft reload on Firefox (@ffdybuster)
 - [#355](https://github.com/pusher/oauth2_proxy/pull/355) Add Client Secret File support for providers that rotate client secret via file system (@pasha-r)
 - [#405](https://github.com/pusher/oauth2_proxy/pull/405) The `/sign_in` page now honors the `rd` query parameter, fixing the redirect after a successful authentication (@ti-mo)
+- [#412](https://github.com/pusher/oauth2_proxy/pull/412) Allow multiple cookie domains to be specified (@edahlseng)
```

Comment:
V5.0.0 has been shipped. Can you shift this up in the changelog

Reply:
This comment still needs to be resolved

Reply:
Done

Reply:
Sorry, this isn't fixed, it needs to be moved up to the new section for changes since v5.1.0, near line 25